sw/device/tock/chips/opentitan_common/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "opentitan_common"
```

Comment:
I think these should just be `opentitan`.

Reply:
Unfortunately the name of board level `opentitan` package clashes with this:
`error: package collision in the lockfile: packages opentitan v0.1.0 (/home/svt/development/repos/opentitan/sw/device/tock/boards/opentitan) and opentitan v0.1.0 (/home/svt/development/repos/opentitan/sw/device/tock/chips/opentitan) are different, but only one can be written to lockfile unambiguously`

Meanwhile I will just change the name of the board level name to "opentitan_board", nothing depends on a top level package at the moment, and we can always sort it out in a follow-up.

Reply:
I'd say let's leave this as is for now and we can change it later if it grates on us too much. 😉